llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/Passes/Passes.h

```diff
@@ -93,8 +93,8 @@ struct MaterializeEncodingInfo {
   SmallVector<int64_t> innerTileSizes;
   SmallVector<int64_t> outerDimsPerm;
 };
-using MaterializeEncodingFn =
-    std::function<FailureOr<MaterializeEncodingInfo>(TensorEncoding, Type)>;
+using MaterializeEncodingFn = std::function<FailureOr<MaterializeEncodingInfo>(
```

Comment:
Hmmm, then it maybe easier to just pass the `ShapedType`. AFAICS, the `ShapedType` already has the encoding

Reply:
Good catch! I was kind of wondering. Done.